Add description and screenshots to manifest article

beaufortfrancois opened the pull request:
This PR adds standardized`description` ~, `categories`,~ and `screenshots` properties to the manifest article so that web developers interested in playing with the PWA install bottom sheet have some documentation.

Note that it currently requires `chrome://flags/#mobile-pwa-install-use-bottom-sheet` flag in Chrome Canary on Android.

Live preview: https://deploy-preview-4502--web-dev-staging.netlify.app/add-manifest/

@petele Can you review?

mihajlija commented on src/site/content/en/progressive-web-apps/add-manifest/index.md:
```diff
@@ -212,6 +225,37 @@ The `shortcuts` property is an array of [app shortcut](/app-shortcuts) objects
+
+#### `categories` {: #categories }
+
+The `categories` property is an array of lowercase strings that describes the
+application categories to which your app belongs.
```
```suggestion
The `categories` property is an array of lowercase strings that describes the
application categories your app belongs to.
```

mihajlija commented on src/site/content/en/progressive-web-apps/add-manifest/index.md:
```diff
@@ -212,6 +225,37 @@ The `shortcuts` property is an array of [app shortcut](/app-shortcuts) objects
+image formats are supported.
+
+{% Aside 'gotchas' %}
+The `categories`, `description` and `screenshots` properties are used only on
+Android for now when a user wants to install your app.
```
```suggestion
The `categories`, `description` and `screenshots` properties are for now used only on
Android when a user wants to install your app.
```

mihajlija reviewed (approved):
LGTM on the writing, feel free to merge after Pete's review.

petele reviewed (approved):
Some minor nits.

petele commented on src/site/content/en/progressive-web-apps/add-manifest/index.md:
```diff
@@ -212,6 +225,37 @@ The `shortcuts` property is an array of [app shortcut](/app-shortcuts) objects
+image formats are supported.
+
+{% Aside 'gotchas' %}
+The `categories`, `description` and `screenshots` properties are for now used
+only on Android when a user wants to install your app.
```
We should probably be more clear that this is only when you've got the flag flipped.

beaufortfrancois replied:
Done.

beaufortfrancois commented on src/site/content/en/progressive-web-apps/add-manifest/index.md:
```diff
@@ -212,6 +225,37 @@ The `shortcuts` property is an array of [app shortcut](/app-shortcuts) objects
+
+#### `categories` {: #categories }
+
+The `categories` property is an array of lowercase strings that describes the
+application categories your app belongs to.
```
Done.

beaufortfrancois commented on src/site/content/en/progressive-web-apps/add-manifest/index.md:
```diff
@@ -212,6 +225,37 @@ The `shortcuts` property is an array of [app shortcut](/app-shortcuts) objects
+The `categories` property is an array of lowercase strings that describes the
+application categories your app belongs to.
+
+The current list of known categories includes books, business, donations,
+education, entertainment, finance, fitness, food, fundraising, games,
+government, health, kids, lifestyle, magazines, medical, music, navigation,
+news, personalization, photo, politics, productivity, security, shopping,
+social, sports, travel, utilities, and weather. To add more categories, file a
+bug to the [manifest-app-info
+repository](https://github.com/w3c/manifest-app-info/issues/new).
```
Done with https://w3c.github.io/manifest-app-info/#categories-member instead of https://www.w3.org/TR/manifest-app-info/#categories-member

beaufortfrancois commented on src/site/content/en/progressive-web-apps/add-manifest/index.md:
```diff
@@ -212,6 +225,37 @@ The `shortcuts` property is an array of [app shortcut](/app-shortcuts) objects
+in common usage scenarios. Each object must include the `src`, a `sizes`
+property, and the `type` of image.
+
+In Chrome, the image dimensions must be at least 320px, and only JPEG and PNG
```
Both

petele replied:
Might be good to call that out explicitly.

beaufortfrancois commented:
> Some minor nits.

I've addressed your feedback at https://github.com/GoogleChrome/web.dev/pull/4502/commits/d084092d60100fea152311122f4ed6a20b3451fa.

beaufortfrancois commented:
@mihajlija I've updated article based on @petele feedback. Feel free to have a look at https://github.com/GoogleChrome/web.dev/commit/d084092d60100fea152311122f4ed6a20b3451fa

beaufortfrancois commented:
I've just updated PR with https://github.com/GoogleChrome/web.dev/pull/4502/commits/7ffc26bf016207e39b327737c7e6a61f5c600010. Still OK?
